iib/workers/tasks/utils.py

```diff
@@ -799,7 +757,7 @@ def run_cmd(
         log.error('The command "%s" failed with: %s', ' '.join(cmd), response.stderr)
         regex: str
         match: Optional[re.Match]
-        if cmd[0] == 'opm':
+        if cmd[0].startswith('opm'):
```

Comment:
Are we assuming the opm binary will be in a directory that's in the PATH? Is something stopping me from specifying `/usr/yashn/opm-1.26.33` in the config? If not, should this be changed to regex that checks for the `opm` string in the value?

Reply:
My assumption was that opm is in $PATH. But you have a good point. 
We should check both - $PATH and if the path set in config is executable opm.

Reply:
sounds valid, I updated in next patch